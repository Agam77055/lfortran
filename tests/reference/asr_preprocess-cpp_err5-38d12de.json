{
    "basename": "asr_preprocess-cpp_err5-38d12de",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/cpp_err5.f90",
    "infile_hash": "3857b759254cc4f4ad48e8255d2825c039fcaf42d8ac223de19565b9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr_preprocess-cpp_err5-38d12de.stderr",
    "stderr_hash": "bbb8f34d4d1571a7b4ee547f7f5f1e0587426e5bdb07789ba5ea8e4b",
    "returncode": 2
}