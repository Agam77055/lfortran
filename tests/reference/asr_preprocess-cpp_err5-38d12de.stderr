C preprocessor error: Unexpected token '^' in factor()
 --> tests/errors/cpp_err5.f90:2:8
  |
2 | #if 1 + ^ 1 2
  |        ^^^ 
